libs/feature-workspaces/src/lib/projects/projects.component.html

```diff
@@ -28,3 +28,23 @@
     </ng-container>
   </mat-list>
 </div>
+
+<ng-template #empty>
+  <ui-dialog>
+    <img src="assets/code-icon.svg" height="180px" />
+    <div class="headline">Your workspace is empty!</div>
+    <div class="sub-headline">
+      Your workspace seems to be empty. Let's create a new Angular application.
+    </div>
+    <button
+      mat-raised-button
+      color="primary"
+      routerLink="../generate?filter=application"
```

Comment:
Mind adding a regression test to make sure this router link doesn't break in the future?

Reply:
done.